get-armory-cli

```diff
@@ -1,5 +1,16 @@
 #! /usr/bin/env bash
 
+throw_error_message() {
+   echo "#############################################################################################################"
+   echo "#                           There was a problem trying to download/install avm, and armory"
```

Comment:
"We're so sorry, we couldn't install Armory Version Manager (avm) or the armory CLI."